compiler/crates/relay-codegen/src/ast.rs

```diff
@@ -156,15 +156,20 @@ pub enum QueryID {
     External(StringKey),
 }
 
-pub struct RequestParameters<'a> {
-    pub id: &'a Option<QueryID>,
+pub enum RequestFormat {
+    ID(QueryID),
+    Text(Option<String>),
+    Migration(QueryID, Option<String>),
```

Comment:
Can we rename this to something more concrete? `TextAndID` maybe?

Reply:
I've simplified the structure to no longer need this additional struct